README.md

```diff
@@ -1 +1,97 @@
 # jeux
+---
+Game night.
+We all love it, we all do it, we all have a special "gamer tag" that defines who we are in that space.
+Ever wanted to try out a game to see if it's worth the price or the time? I bet you have. We have the solution for you!
+Jeux is a handy site that lets you and your friends share games with one another while allow you to keep track of who has what. No more wondering where your Friday night specialty has gone! Jeux has you covered!
+
+## Table of Contents
+---
+- [What's inside (Technologies)](#What's-inside-Technologies)
+- [Meet the team](#Meet-the-team)
+- [File Structure](#File-Structure)
+- [Standards](#Standards)
+- [Deadlines](#Deadlines)
+- [User Stories](#User-Story)
+
+## What's inside (Technologies)
+---
+### Frontend
+* Laravel
+* Php
+* SASS
+* Vue.js
+* OAuth
+* Bootstrap
+### Backend
+* MySql
+* Eloquent
+###Testing gear
+* Phpunit
```

Comment:
do we need to add puppeteer for e2e testing @iverunamok?

Reply:
Yes Please :D will we also list Headless Chrome in here? or would that be considered part of Puppeteer?

Reply:
good question, look it up :)